src/main/java/appeng/client/render/AERenderItem.java

```diff
@@ -0,0 +1,81 @@
+package appeng.client.render;
+
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.renderer.entity.RenderItem;
+
+import org.lwjgl.opengl.GL11;
+
+import appeng.api.config.TerminalFontSize;
+import appeng.util.ISlimReadableNumberConverter;
+import appeng.util.IWideReadableNumberConverter;
+import appeng.util.ReadableNumberConverter;
+
+public abstract class AERenderItem extends RenderItem {
+
+    private static final ISlimReadableNumberConverter SLIM_CONVERTER = ReadableNumberConverter.INSTANCE;
+    private static final IWideReadableNumberConverter WIDE_CONVERTER = ReadableNumberConverter.INSTANCE;
+
+    public void drawStackSize(int offsetX, int offsetY, long stackSize, FontRenderer font, Enum fontSize) {
```

Comment:
Please use specific type (`TerminalFontSize`) where possible instead of `Enum`. Otherwise use wildcard variant `Enum<?>` instead of raw type `Enum`.